scripts/tarchain.ts

```diff
@@ -0,0 +1,74 @@
+import fs from "fs-extra";
+import path from "path";
+import yargs from "yargs";
+import { spawn, SpawnOptions } from "child_process";
+
+yargs
+  .scriptName("tarchain")
+  .recommendCommands()
+  .demandCommand(1)
+  .strict(true)
+  .showHelpOnFail(true)
+  .command(
+    "run <datadir> <monorepo>",
+    "Create a new tarball that includes the deployed staked CELO contracts",
+    (args) =>
+      args
+        .positional("datadir", { type: "string", description: "Path to devchain data directory" })
+        .positional("monorepo", { type: "string", description: "Path to monorepo" })
+        .option("filename", {
+          type: "string",
+          description: "Filename of output tarball (default: stakedCeloDevchain.tar.gz)",
+        }),
+    (args) =>
+      exitOnError(
+        runCmd(
+          args.datadir!,
+          args.filename != ("" || undefined) ? args.filename : "stakedCeloDevchain.tar.gz",
+          args.monorepo!
+        )
+      )
+  ).argv;
+
+function runCmd(datadir: string, filename: string, monorepo: string) {
+  const chainDataDir = "chainData/";
+  const deploymentsDir = "deployments/local";
+
+  fs.copySync(deploymentsDir, chainDataDir + deploymentsDir);
+  const cmdArgs = ["devchain", "compress-chain", datadir, chainDataDir + filename];
+  const ProtocolRoot = path.normalize(path.join(__dirname, `../${monorepo}/packages/protocol`));
```

Comment:
Why is there a `..` before the `monorepo` value? I would assume `monorepo` is meant to point directly at the monorepo already.

Reply:
Used `process.cwd()` instead of `__dirname`. This way `protocolRoot` is based on the `cwd` instead of the location of `tarchain.ts`.

This change should prevent breakage if the `tarchain.ts` get nested further.

Reply:
I guess I don't understand why you're `join`ing anything to the already fully specified monorepo path. E.g. if `cwd` is `/home/martin/staked-celo` and I provide the monorepo path as an absolute path, e.g. `/home/martin/celo-monorepo`, the current code would lead to `/home/martin/staked-celo/home/martin/celo-monorepo`, right? Which doesn't exist and isn't what we want.

Reply:
I joined them since I was using relative path. 

I can change it to using absolute path instead.

Reply:
changed to absolute path

Reply:
I mean, this should now work with either absolute or relative paths. If I do `--monorepo ../<rest of path>` or `--monorepo /<absolute path>`, this should work in either case. Before, it was essentially required you'd provide a relative path, relative to the directory higher up, rather than relative to the current directory. Right?